src/averageAges.js

```diff
@@ -14,12 +14,17 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = 0) {
+  const age = century !== 0
+    ? people
+      .filter((person) => Math.ceil(person.died / 100) === century)
```

Comment:
instead of iterating all people twice check if `century` is defined inside the filter's callback and filter by century too if yes

Reply:
do it in the same way as you do inside find callback in third function